### PR TITLE
Initialized employee subclasses in EmployeeManager

### PR DESCRIPTION
**Pull request by FaithD186**
added employee subclasses to employee_list

### CODE REVIEW

**Review by yi1z — APPROVED** [review]

**Comment by yi1z** [comment]
I resolved the conflicts, but I still need Faith to check if the implementations are still correct and can do the job

**Review by Maggie39 — APPROVED** [review]

**Review by h4kureii — CHANGES_REQUESTED** [review]
Uhhhhh this edition of EmployeeManager is outdated. We don't use addEmployee() anymore, and employees.txt isn't created when it's not found anymore (we just assume it exists now).
We're also moving the text parsing into EmployeeManagerReadWriter, so I think we should hold off on this PR until we merge EmployeeManagerReadWriter, then fix it after.

Friendly reminder to all to regularly update your branches.

**Comment by FaithD186** [comment]
> Uhhhhh this edition of EmployeeManager is outdated. We don't use addEmployee() anymore, and employees.txt isn't created when it's not found anymore (we just assume it exists now). We're also moving the text parsing into EmployeeManagerReadWriter, so I think we should hold off on this PR until we merge EmployeeManagerReadWriter, then fix it after.
Ok, I'll fix this after we merge EmployeeManagerReadWriter

**Pull request closed without merging**
